ibc/modules/README.md

```diff
@@ -1,47 +1,212 @@
 # IBC module
 
-[![Crate][crate-image]][crate-link]
-[![Docs][docs-image]][docs-link]
-[![Build Status][build-image]][build-link]
-[![End to End testing][e2e-image]][e2e-link]
 [![Apache 2.0 Licensed][license-image]][license-link]
-![Rust Stable][rustc-image]
-![Rust 1.51+][rustc-version]
 
+Implementation of the Inter-Blockchain Communication Protocol ([IBC]) in rust.
```

Comment:
Definitely link the other sub docs to this one and move them all to a `docs/` subfolder